recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-core/packagegroups/packagegroup-xt-core-xen.bb

```diff
@@ -6,5 +6,4 @@ inherit packagegroup
 
 RDEPENDS_packagegroup-xt-core-xen = "\
     xen \
-    xen-tools \
```

Comment:
Are you sure that we can remove that?

Reply:
In this variable there was xen-base, and by mistake I added xen-tools instead of xen. I thought xen-base is replaced by xen-tools because recipe xen-base was deleted and xen-tools was added in meta-virtualization.
I ask the Oleksandr Andrushchenko and he confirmed that xen-tools are only needed by backends and userspace. 
In this package it is redundant
I have started new build with latest changes to verify it. I will notify as soon as it is ready.

Reply:
ok, I am fine with that.